packages/console/src/browser/style/index.css

```diff
@@ -22,11 +22,15 @@
 
 .theia-console-input {
     padding-left: 20px;
-	border-top: var(--theia-panel-border-width) solid var(--theia-panel-border);
+    border-top: var(--theia-panel-border-width) solid var(--theia-panel-border);
+    height: 45px;
```

Comment:
why 45px? @vince-fugnitto maybe we have some css variable which we can use to compute height instead

hardcoded constants are not good, they don't scale when we change variable values

Reply:
I agree, it's one of the reasons I did not like my initial attempt to resolve the issue.
@kaiyue0329 we should try a css variable that we can re-use to compute the height.